demo/src/main/res/values/strings.xml

```diff
@@ -0,0 +1 @@
+<resources></resources>
```

Comment:
Erroneous file addition?

Reply:
Yeah this one not sure but it was probably generated. Deleted.